x-pack/test_serverless/README.md

```diff
@@ -133,3 +133,21 @@ node scripts/functional_tests_server.js --config test_serverless/api_integration
 
 node scripts/functional_test_runner.js --config test_serverless/api_integration/test_suites/search/config.ts
 ```
+
+## Skipping tests for MKI run
+The tests that are listed in the the regular `config.ts` generally should work in both Kibana CI and MKI. However some tests might not work properly against MKI projects by design.
+Tag the tests with `skipMKI` to be excluded for MKI run. It works only for the `describe` block:
+```
+describe("my test suite", async function() {
+    this.tags(['skipMKI']);
+    ...
+});
+```
+
+If you are running tests against MKI locally, make sure to add `--exclude-tag=skipMKI` to your FTR command.
```

Comment:
I'd suggest to slightly reword this as I find the phrase `running tests against MKI locally` a bit confusing. Maybe something like this?
```
If you are running tests from your local against MKI projects, make sure to add `--exclude-tag=skipMKI` to your FTR command.
```